### ISSUE TITLE
Error: Getter not found: 'obscuringCharacter'.

### ISSUE THREAD

**Issue by imrhk** [matthinc/flutter_cupertino_settings#20]
```
Compiler message:
../../../.pub-cache/hosted/pub.dartlang.org/flutter_cupertino_settings-0.3.0/lib/widgets/secret.dart:41:58: Error: Getter not found: 'obscuringCharacter'.
                  _show ? widget.secret : RenderEditable.obscuringCharacter * widget.secret.length,
```

I am getting this error after updating flutter.

flutter doctor -v output
```
Rahuls-MacBook-Pro-2:app rahul$ flutter doctor -v
[✓] Flutter (Channel beta, 1.18.0-11.1.pre, on Mac OS X 10.15.2 19C57, locale en-IN)
    • Flutter version 1.18.0-11.1.pre at /Users/rahul/Library/flutter/flutter
    • Framework revision 2738a1148b (3 days ago), 2020-05-13 15:24:36 -0700
    • Engine revision ef9215ceb2
    • Dart version 2.9.0 (build 2.9.0-8.2.beta)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/rahul/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/rahul/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] Android Studio
    • Android Studio at /Applications/Android Studio 4.0 Preview.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] IntelliJ IDEA Community Edition (version 2019.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.45.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.10.2

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • E90AE266-0001-4DB6-A49E-1FA447C4FEFB • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)

! Doctor found issues in 2 categories.
```

**Comment by imrhk** [comment]
_Flutter Beta Channel_
https://github.com/flutter/flutter/blob/beta/packages/flutter/lib/src/rendering/editable.dart#L350
Previously this was static method. now it's a member function of `RenderEditable` class. Please make change for the same.

_Flutter Stable Channel_
https://github.com/flutter/flutter/blob/stable/packages/flutter/lib/src/rendering/editable.dart#L282

Clearly the issue is with beta channel.

Thanks,

**Comment by codegrue** [comment]
Same. This error is on the latest beta channel `Flutter 1.18.0-11.1.pre • channel beta`

**Comment by matthinc** [comment]
Thank you for the issue.
Should be fixed with ee904c2f20f8985761b7794d058c26c5a2bb743d

**Comment by matthinc** [comment]
Fixed in 0.3.1
